laravel/Caddyfile

```diff
@@ -0,0 +1,17 @@
+example.com {
+
+    root ./public
+    log ./storage/logs/caddy-access.log
+    errors ./storage/logs/caddy-error.log
```

Comment:
@mholt I agree that `log` & `errors` are not required. All laravel apps default to "public" as the root/"public" folder though.